src/components/virtualized-select.tsx

```diff
@@ -0,0 +1,95 @@
+import { memo, useState } from 'react';
+import Async, { AsyncProps } from 'react-select/async';
+import { MenuListProps, StylesConfig } from 'react-select';
+import { FixedSizeList } from 'react-window';
+import { SelectComponents } from 'react-select/dist/declarations/src/components';
+
+type VirtualizedSelectProps = AsyncProps<any, any, any> & {
```

Comment:
Same comment that I left for search-bar.tsx about separating `AsyncProps` and your component props.

Reply:
I think this one might be fine as the only component I am returning in this component is the Async itself. It is just modifying it. I have changed it in the search-bar.js as suggested though.